README.md

```diff
@@ -27,6 +30,37 @@ gitmoji-changelog
 
 **// TODO**
 
+## Setup
+
+Install the lib in your PATH : 
+
+```bash
+npm install -g gitmoji-changelog
+```
+
+## Mapping
+
+When a commit is done with a gitmoji, the following mapping is used to group them : 
+
+| Categorie | Gitmoji |
+|---|---|
+| Added | :sparkles: :tada: :white_check_mark: :bookmark: :construction_worker: :charts_with_upwards_trend: :heavy_plus_sign: :loud_sound: |
+| Changed | :art: :zap: :lipstick: :rotating_light: :arrow_down: :arrow_up: :pushpin: :recycle: :wrench: :rewind: :alien: :truck: :boom: :bento: :wheelchair: :speech_ballon: :card_file_box: :children_crossing: :building_construction: :iphone: |
```

Comment:
Here too `speech_balloon` 😉

Reply:
Ok, so there is a typo in `mapping.js`, I'll make another PR with this correction